cms/static/js/spec/views/video_thumbnail_spec.js

```diff
@@ -149,38 +195,152 @@ define(
                 videoThumbnailView.chooseFile();
 
                 // Add image to upload queue and send POST request to upload image
-                $el.find('.upload-image-input').fileupload('add', {files: [createFakeImageFile(60)]});
+                $el.find('.upload-image-input').fileupload('add', {files: [createFakeImageFile()]});
 
                 AjaxHelpers.respondWithError(requests, 400);
 
                 verifyStateInfo($thumbnail, 'error');
             });
 
-            it('should show error notification in case of server error', function() {
-                var $el = render({}),
+            it('calls readMessage with correct message', function() {
+                var successData = {
+                        files: [createFakeImageFile()],
+                        submit: function() {}
+                    },
+                    failureData = {
+                        jqXHR: {
+                            responseText: JSON.stringify({
+                                error: 'The selected image contains unsupported image file type.'
```

Comment:
This image file type is not supported. Supported file types are .gif, .jpg, and .png.